app/jobs/webhooks/facebook_events_job.rb

```diff
@@ -1,8 +1,28 @@
 class Webhooks::FacebookEventsJob < ApplicationJob
+  class LockAcquisitionError < StandardError; end
+
   queue_as :default
+  # https://edgeapi.rubyonrails.org/classes/ActiveJob/Exceptions/ClassMethods.html#method-i-retry_on
+  retry_on LockAcquisitionError, wait: 10.seconds, attempts: 5
```

Comment:
The timing looks okay. We could even reduce the timing to, say `2` seconds.  The process message job should be done within a sub-second.  the more time here, it would feel like subsequent messages to be lagging for the end user